### PR TITLE
Use go1.16 embedded files for graphql schema

### PR DESCRIPTION
**Pull request by eseliger**
I think this is pretty neat as it can never be forgotten to regenerate and such. Also, it will enable a workflow of changing the schema through a github pr suggestion, without the need to regenerate locally.

This requires https://github.com/sourcegraph/sourcegraph/pull/18765 to pass the licenses check.

### CODE REVIEW

File: dev/handle-change.sh [hunk 1]
```diff
@@ -13,7 +12,7 @@ failed=false
 for i; do
   case $i in
     "cmd/frontend/graphqlbackend/schema.graphql")
-      generate_graphql=true
+      cmdlist+=("frontend")
```
Comment (eseliger) [inline dev/handle-change.sh:15]: We just assume that frontend needs to be rebuilt when the schema changes, and add it to the list.

**Review by slimsag — APPROVED** [review]

**Review by LawnGnome — APPROVED** [review]
image

File: dev/handle-change.sh (hunk 1 above)
Comment (keegancsmith) [reply dev/handle-change.sh:15]: Maybe this should be a comment in the code :)

**Review by keegancsmith — APPROVED** [review]

**Comment by ryanslade** [comment]
image

Beautiful 😄

File: dev/handle-change.sh (hunk 1 above)
Comment (eseliger) [inline dev/handle-change.sh:15]:
```suggestion
      # We assume that frontend needs to be rebuilt when the schema changes, and add it to the list.
      cmdlist+=("frontend")
```

**Pull request merged**